src/rpc/server/basic.zig

```diff
@@ -387,97 +387,72 @@ fn handleRpcRequest(
         break :json result;
     };
 
-    switch (rpc_request.method) {
-        .getAccountInfo => |params| {
-            const config: rpc.methods.GetAccountInfo.Config = params.config orelse .{};
-            const encoding = config.encoding orelse .base64;
-            if (config.commitment) |commitment| {
-                std.debug.panic("TODO: handle commitment={s}", .{@tagName(commitment)});
-            }
+    switch (@as(rpc.methods.MethodAndParams.Tag, rpc_request.method)) {
+        inline else => |method| {
+            const allocator = json_arena;
+            const result = blk: {
+                // Avoid eager-eval of any inline-else method to call()'s rcp.methods.Request().
```

Comment:
Could make that `Request` function return a `?type`, or a variant of it that `Request` calls internally, in order to avoid the noreturn compile error; would make it easier to update in the future when we've finally implemented all the methods there and remove the optional-ness to find all these sites.

Edit: does calling it actually cause a compile error? I would have expected the fact that the compile-erroring tags are `noreturn` to make the `inline else => |method|` here to never return eval to any of those anyway, since they're unreachable branches.